allennlp/training/callbacks/confidence_checks.py

```diff
@@ -2,25 +2,25 @@
 
 from allennlp.training.callbacks.callback import TrainerCallback
 from allennlp.data import TensorDict
-from allennlp.sanity_checks.normalization_bias_verification import NormalizationBiasVerification
+from allennlp.confidence_checks.normalization_bias_verification import NormalizationBiasVerification
 
 
 if TYPE_CHECKING:
     from allennlp.training.trainer import GradientDescentTrainer
 
 
-@TrainerCallback.register("sanity_checks")
-class SanityChecksCallback(TrainerCallback):
+@TrainerCallback.register("confidence_checks")
```

Comment:
Maybe should we should keep this also registered as "sanity_checks" for now?

```suggestion
@TrainerCallback.register("confidence_checks")
@TrainerCallback.register("sanity_checks")
```